.github/workflows/api_test.yaml

```
@@ -0,0 +1,22 @@
+name: API Test
+on: 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)